src/settings.ts

```diff
@@ -0,0 +1,34 @@
+/**
+ * The resolved and normalized settings for this extension, the result of calling resolveSettings on a raw settings
+ * value.
+ *
+ * See the "contributes.configuration" field in package.json for the canonical documentation on these properties.
+ */
+export interface Settings {
+    ['sentry.organization']?: string
+    ['sentry.projects']?: [
+        {
+            name: string
+            projectId: string
+            patternProperties: {
+                repoMatch: RegExp
```

Comment:
Just checking - you have yet to write the code that actually resolves the string values from the settings to `RegExp`, right?

Reply:
No, I already do that in `settings`.